app/src/docs/_examples/tables/TablesExamples.module.scss

```diff
@@ -30,3 +30,11 @@
 .weekend-day {
     background-color: #EBEDF5;
 }
+
+.action-button {
+    padding: 0 20px;
```

Comment:
why we need to add custom styles here? I supposed that we should use DropdownMenuButton without any additional customization

Reply:
removed